Index: src/main/java/ru/job4j/concurrent/ConcurrentOutput.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/ru/job4j/concurrent/ConcurrentOutput.java b/src/main/java/ru/job4j/concurrent/ConcurrentOutput.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/ru/job4j/concurrent/ConcurrentOutput.java	
@@ -0,0 +1,15 @@
+package ru.job4j.concurrent;
+
+public class ConcurrentOutput {
+    public static void main(String[] args) {
+        Thread another = new Thread(
+                () -> System.out.println(Thread.currentThread().getName())
+        );
+        another.start();
+        System.out.println(Thread.currentThread().getName());
+        Thread second = new Thread(
+                () -> System.out.println(Thread.currentThread().getName())
+        );
+        second.start();
+    }
+}
\ No newline at end of file
